Dockerfile

```diff
@@ -25,7 +25,8 @@ ENV ALLOW_RESTARTS=0 \
     SYSTEM=0 \
     TASKS=0 \
     VERSION=1 \
-    VOLUMES=0
+    VOLUMES=0 \
+    LOG_LEVEL=info
```

Comment:
Please sort it alphabetically.

Reply:
I store messages in Elasticsearch. 70% of the messages are docker-socket-proxy. I don't need to store all calls. I hope to save a lot of space by being able to limit the logs.

I push a new commit with variable alphabetically sorted.